### PR TITLE
Add new wrappers functions for IP Set using VDC Groups

### PR DESCRIPTION
**Pull request by vbauzysvmware**
Changes for terraform - new functions added:
* VdcGroup.GetNsxtFirewallGroupById
* VdcGroup.GetNsxtFirewallGroupByName

Test with 10.3.1 and 10.2.2

### CODE REVIEW

File: govcd/nsxt_firewall_group.go [hunk 1]
```diff
@@ -124,6 +124,27 @@ func (vdc *Vdc) GetNsxtFirewallGroupByName(name, firewallGroupType string) (*Nsx
 	return getNsxtFirewallGroupByName(vdc.client, name, queryParameters)
 }
 
+// GetNsxtFirewallGroupByName allows users to retrieve Firewall Group by Name in a particular Vdc Group
```
Comment (lvirbalas) [inline govcd/nsxt_firewall_group.go:127]:
```suggestion
// GetNsxtFirewallGroupByName allows users to retrieve Firewall Group by Name in a particular VDC Group
```

**Review by lvirbalas — APPROVED** [review]
Nice to see a small PR for a change :) Just one ask.

File: govcd/nsxt_firewall_group.go (hunk 1 above)
Comment (vbauzysvmware) [reply govcd/nsxt_firewall_group.go]: done

**Review by adambarreiro — APPROVED** [review]
LGTM! 👍

**Review by Didainius — APPROVED** [review]
Thanks - tests worked on 10.2.0 and 10.3.2

**Review by mikeletux — APPROVED** [review]
LGTM! Approved!

**Pull request merged**